modules/cnf-running-the-performance-creator-profile.adoc

```diff
@@ -0,0 +1,197 @@
+      --user-level-networking             Run with User level Networking(DPDK) enabled
+----
+
+. Run the performance profile creator tool in discovery mode:
++
+[NOTE]
+====
+Discovery mode inspects your cluster using the output from `must-gather`. The output produced includes information on the:
+
+* NUMA cell partitioning with the allocated CPU ids
+* Whether hyperthreading is enabled
+
+Using this information you can set appropriate values for some of the arguments supplied to the performance profile creator tool.
+====
++
+[source,terminal]
+----
+$ podman run --entrypoint performance-profile-creator -v /must-gather:/must-gather:z quay.io/openshift-kni/performance-addon-operator:4.8-snapshot --info log --must-gather-dir-path /must-gather
+----
++
+[NOTE]
+====
+This command uses the performance profile creator as a new entry point to `podman`. It maps the `must-gather` data for the host into the container image and invokes the required user-supplied profile arguments to produce the `my-performance-profile.yaml` file.
+
+The `-v` option can be the path to either:
+
+* The `must-gather` output directory
+* An existing directory containing the `must-gather` decompressed tarball
+
+The `info` option requires a value which specifies the output format. Possible values are log and JSON. The JSON format is reserved for debugging.
+====
++
+. Run `podman`:
++
+[source,terminal]
+----
+$ podman run --entrypoint performance-profile-creator -v /must-gather:/must-gather:z quay.io/openshift-kni/performance-addon-operator:4.8-snapshot --mcp-name=worker-cnf --reserved-cpu-count=20 --rt-kernel=true --split-reserved-cpus-across-numa=false --topology-manager-policy=single-numa-node --must-gather-dir-path /must-gather  --power-consumption-mode=ultra-low-latency > my-performance-profile.yaml
+----
++
+The following arguments are required:
+
+* `reserved-cpu-count`
+* `mcp-name`
++
+The `mcp-name` argument in this example is set to `worker-cnf` based on the output of the command `oc get mcp`.
+* `rt-kernel`
++
+[NOTE]
+====
+For Single Node OpenShift (SNO) use `--mcp-name=master`.
+====
+
++
+. Review the created YAML file:
++
+[source,terminal]
+----
+$ cat my-performance-profile.yaml
+----
+.Example output
++
+[source,terminal]
+----
+apiVersion: performance.openshift.io/v2
+kind: PerformanceProfile
+metadata:
+  name: performance
+spec:
+  additionalKernelArgs:
+  - nmi_watchdog=0
+  - audit=0
+  - mce=off
+  - processor.max_cstate=1
+  - intel_idle.max_cstate=0
+  - idle=poll
+  cpu:
+    isolated: 1,3,5,7,9,11,13,15,17,19-39,41,43,45,47,49,51,53,55,57,59-79
+    reserved: 0,2,4,6,8,10,12,14,16,18,40,42,44,46,48,50,52,54,56,58
+  nodeSelector:
+    node-role.kubernetes.io/worker-cnf: ""
+  numa:
+    topologyPolicy: single-numa-node
+  realTimeKernel:
+    enabled: true
+----
+
+. Apply the generated profile:
+
++
+[source,terminal]
+----
+$ oc apply -f my-performance-profile.yaml
+----
+
+.How to run `podman` to create a Performance profile
```

Comment:
This .Title type of heading only gives a heading to the paragraph below it. It's not really a proper section heading. If you want to make this part below a seperate subsection, then I'd suggest using a discrete heading:

```
[discrete]
How to run `podman` to create a Performance profile
```

However, putting it at the end of a procedure module probably isn't the right place for it. It should probably be it's own module and included from the assembly.

Reply:
Also, "Performance profile" looks to be lowercase elsewhere - should be consistent.